content/en/apps/tutorials/localizing-cht.md

```diff
@@ -0,0 +1,169 @@
+
+e.g to change the date of birth field to Swahili, in `messages-sw.properties` file, add this:
+
+```
+contact.type.date_of_birth = Siku ya Kuzaliwa
+```
+![configuration](localize-reports.png)
+
+
+e.g 
+
+_**People**_
+To localize the contact labels, add the appropriate translation in `messages-{language-code}.properties` file.
+
+e.g to change the chv name field to Swahili, in `messages-sw.properties` file, add this:
+
+```
+contact.type.district_hospital = Kituo Cha Afya
+```
+
+_**Targets**_
+You can localize the names of the targets by adding the appropriate translation in the `messages-{language-code}.properties file.
+
+e.g to change the `Number of fever cases managed` target to Swahili on the instance, adding the appropriate translation in the `messages-sw.properties` file. i.e
+
+```
+targets.fever_cases_managed.title = Idadi ya kesi za homa kusimamiwa
+```
+
+#### 2. App Forms
+To localize an app form to say the Kiswahili language, open the appropriate xlsx of the form and add a `label::sw` column which has the translation for the text. This will work in the `Survey` sheet or the `choices` sheet.
+
+![configuration](app-forms-localization.png)
+
+
+_**App Management - Admin Area**_
+This will still remain in English even after changing the default language.
+
+
+
+**Note**: The CHV can also choose the language of their choice when they login for the first time. A popup appears where they can choose their preferred language. 
+### 3. Outgoing Texts
+
+These are the SMS notifications/replies that go to CHVs and Supervisors phones in projects that incorporate SMS workflows.
+
+To change the language of outgoing texts to a particular CHV/Supervisor to Swahili: 
+
+i). First in app settings, when configuring replies, add Swahili(sw) under `locales`. Like so:
+```
+"locales": [
+    {
+      "code": "en",
+      "name": "English"
+    },
+    {
+      "code": "es",
+      "name": "Spanish"
+    },
+    {
+      "code": "fr",
+      "name": "French"
+    },
+    {
+      "code": "ne",
+      "name": "Nepali"
+    },
+    {
+      "code": "sw",
+      "name": "Swahili"
+    }
+``` 
+
+ii) Set up the translation for the reply message. For example:
+```
+"messages": [
+        {
+          "message": [
+            {
+              "content": "Thank you, visit for {{patient_name}} ({{patient_id}}) has been recorded.",
+              "locale": "en"
+            },
+            {
+              "content": "Asante, kuhudhuria kwa {{patient_name}} ({{patient_id}} kumerekodiwa.",
+              "locale": "sw"
+            }
+          ],
+          "event_type": "report_accepted",
+          "recipient": "reporting_unit"
+        },
+``` 
+iii) Change the CHVs language in app management > users > [Choose CHV username e.g chv_1] > Language 
+
+![configuration](change-user-language.png)
+
+### Translating CHT to another language
```

Comment:
Is there a plan to add steps to translate targets?

Reply:
https://github.com/medic/cht-docs/blob/31a32b05f75020c2686b18dc7a2a8f5f90db5b1b/content/en/apps/tutorials/localizing-cht.md?plain=1#L101